index.html

```diff
@@ -0,0 +1,111 @@
+<!--Adopt-a-Dog Main Page 1 -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt-a-Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!--this is for the future style sheet-->
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+        <img src="adoptadog-logo.svg" width="500" alt="white dog silhoutte">
+        <h2>Seattle's Premier Dog Adoption Agency</h2>
+        <ing src="cart-icon.svg" width="200" alt="cart icon">
+          <a href="dog-checkout-page4.html">Checkout</a>
+      </header>
+      <nav>
+        <ul>
+          <li>Dogs</li>
+          <li>Blog</li>
+          <li>Adopt</li>
+        </ul>
+      </nav>
+      <div>
+        <img src="home-intro.jpg" width="500" alt="a boy hugs a happy Golden Retriver">
+          <h2> Adopt a dog. </h2>
+            <p>Puppy kitty ipsum dolor sit good dog foot stick canary. Teeth Mittens grooming vaccine walk swimming nest good boy furry tongue heel furry treats fish. Cage run fast kitten dinnertime ball run foot park fleas throw house train licks stick dinnertime window. Yawn litter fish yawn toy pet gate throw Buddy kitty wag tail ball groom crate ferret heel wet nose Rover toys pet supplies. Bird Food treats tongue lick teeth ferret litter box slobbery litter box crate bird small animals yawn small animals shake slobber gimme five toys polydactyl meow.
+            </p>
+        </div>
```

Comment:
make sure your indenting is right. the p gat should be the same level as the h2 tag, it gets a little shaggy quick